starknet_py/net/utils.py

```diff
@@ -0,0 +1,84 @@
+from collections import OrderedDict
```

Comment:
Account related utils should be in `/account`, not in top level.

Reply:
Preparing an invoke transaction is not Account related, is it?

Reply:
(A dummy one, from arguments, without the signature)